README.md

```diff
@@ -14,6 +14,21 @@ A repository for hosting nextflow [`DSL2`](https://www.nextflow.io/docs/edge/dsl
     * [Uploading to `nf-core/modules`](#uploading-to-nf-coremodules)
 * [Help](#help)
 
+## Terminology
+
+The DSLv2 features for Nextflow are new for everyone and not just beginners. We discussed some terminology terms to discuss things related to modules & subworkflows more appropriately:
+
+* *Module*: An "atomic" module (one thing), e.g. a reusable process from an old pipeline that can be used between different pipelines
+* *Subworkflow*: A combined set of modules, that do e.g. preprocessing for a much bigger pipeline. A good example could be a QC subworkflow for FastQ input, that could be (re-) used across multiple pipelines
```

Comment:
* *Subworkflow*: A set of modules combined using `workflow <name>`, that performs a logical stage within a bigger pipeline, e.g., a quality control subworkflow for FastQ input.